src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -181,6 +189,14 @@ def register_options(cls, register):
       ]
     )
 
+  @memoized_property
+  def _rsc_native_image(self):
+    return RscNativeImage.scoped_instance(self)
+
+  @memoized_property
+  def _rsc_classpath(self):
+    return self.tool_classpath('rsc')
```

Comment:
This change probably isn't necessary -- it was useful for a previous iteration in which I attempted to make `Rsc` into its own subsystem like `Zinc`, but I had a lot of difficulty with that so kept it just to an `RscNativeImage` subsystem.

Reply:
Harmless... can leave it for now.